react/features/toolbox/functions.native.js

```diff
@@ -10,8 +10,9 @@ import { toState } from '../base/redux';
  * @returns {boolean}
  */
 export function isToolboxVisible(stateful: Object | Function) {
-    const { alwaysVisible, enabled, visible }
-        = toState(stateful)['features/toolbox'];
+    const state = toState(stateful);
+    const { alwaysVisible, enabled, visible } = state['features/toolbox'];
```

Comment:
What controls the 'visible' flag now? Is it being adjusted anywhere on mobile? Does it default to `true`?

Reply:
The visible flag is controlled with the tap anywhere on the conference. This adds the 1 participant exception thing to simplify a bit. I got insipired by the filmstrip function while I was there.